scripts/uberenv/packages/serac/package.py

```diff
@@ -97,24 +96,31 @@ class Serac(CMakePackage):
     depends_on('py-sphinx', when="+devtools")
 
     # Libraries that support +debug
-    depends_on("mfem~shared~zlib+hypre+metis+superlu-dist+lapack+mpi")
-    depends_on("mfem~shared~zlib+hypre+metis+superlu-dist+lapack+mpi+debug", when="+debug")
-    depends_on("hypre~shared~superlu-dist+mpi")
-    depends_on("hypre~shared~superlu-dist+mpi+debug", when="+debug")
+    debug_deps = ["mfem@4.0.0~shared~zlib+hypre+metis+superlu-dist+lapack+mpi",
+                  "hypre@2.11.1~shared~superlu-dist+mpi"]
+    for dep in debug_deps:
+        depends_on("{0}".format(dep))
+        depends_on("{0}+debug".format(dep), when="+debug")
+
+    depends_on("hypre@2.11.1~shared~superlu-dist+mpi")
```

Comment:
Not sure how i mess this up but these two lines need to be removed because they are specified in the for loop above.

Reply:
Yup, also present in #66, but I. should have tracked it done to this PR: https://github.com/LLNL/serac/pull/66#discussion_r436901597